veocean-data-farming/df-intro.md

```diff
@@ -0,0 +1,91 @@
+---
+description: An introduction to Data Farming and Ocean Protocol's key incentives.
+---
+# Data Farming 101
+
+Data Farming (DF) incentivizes for growth of Data Consume Volume (DCV) in the Ocean ecosystem.
+
+It rewards OCEAN to liquidity providers (stakers) as a function of consume volume and liquidity. It’s like DeFi liquidity mining, but tuned for data consumption. DF’s aim is to achieve a minimum supply of data for network effects to kick in, and once the network flywheel is spinning, to increase growth rate.
+
+## Reward Categories
+Rewards are paid in OCEAN and distributed every week on Thursday as follow:
+
+| Passive Rewards | Active Rewards |
+| ---- | ------- |
+| 50%    | 50%    |
+
+Active Rewards are governed and defined by the [Reward Function](df-background.md#reward-function).
+
+**Final Caveat:** We reserve the right to make reasonable changes to these plans, if unforeseen circumstances emerge.
+
+## How to access DF and claim rewards
+
+Please [follow this tutorial](../rewards/veOcean-Data-Farming-Tutorial.md) to learn how the Ocean Protocol reward programs work, and how to access them.
+
+Otherwise, go to the DF webapp at [df.oceandao.org](df.oceandao.org/) and explore Data Farming for yourself.
+
+### Where to claim?  
+All earnings for veOCEAN holders are claimable on the ”Rewards” page inside the Data Farming webapp on Ethereum mainnet.  
+
+Data assets for DF may published in any [network where Ocean’s deployed in production](../core-concepts/networks.md): Eth mainnet, Polygon, etc.
+
+### When to claim?
+There are fresh rewards available every Thursday. If you wish, you can wait for many weeks to accumulate before claiming. (It’s all on-chain.)
+
+### When to do a first claim?
+From the time you lock OCEAN, you must wait at least a week, and up to two weeks, to be able to claim rewards.  
+
+The nerdy version: if you lock OCEAN on day x, you’ll be able to claim rewards on the first weekly ve “epoch” that begins after day x+7.  
+
+This behavior is inherited from [veCRV](https://curve.readthedocs.io/dao-fees.html); [here’s the code](https://github.com/oceanprotocol/contracts/blob/main/contracts/ve/veFeeDistributor.vy#L240-L256).
+
+## DF Main
+
+DF Main started Mar 16, 2023 in DF Round 29. DF29 has 150K OCEAN rewards available (a 2x increase from DF28). As DF Main progresses, rewards will increase to 300K (another 2x), then 600K (another 2x), then beyond 1.1M OCEAN/week (near 2x) then decaying over time.
+
+As of DF29, wash consuming will no longer be profitable. So, organically-generated Data Consume Volume will be the main driver of active DF rewards.
```

Comment:
This time has passed. So need to rewrite for the current. E.g.:

As of DF29 (Mar 16, 2023), wash consuming is not profitable. So, organically-generated Data Consume Volume is the main driver of active DF rewards.

Reply:
I overlooked this one, thank you. :pray: